build-faq.md

```diff
@@ -4,4 +4,58 @@ When building on macOS, the build system will make a native Cocoa app by default
 
 # Attempting to build the SDL frontend on macOS fails on linking
 
-SameBoy on macOS expects you to have SDL2 installed via Brew, and not as a framework. Older versions expected it to be installed as a framework, but this is no longer the case.
\ No newline at end of file
+SameBoy on macOS expects you to have SDL2 installed via Brew, and not as a framework. Older versions expected it to be installed as a framework, but this is no longer the case.
+
+# Windows build process
+
+For the various tools and libraries, follow the below guide to ensure easy, proper configuration for the build environment:
+
+#### clang
+
+This may be installed via a Visual Studio installer packages instead of built from source.
+
+#### SDL Port
+
+[libsdl2](https://libsdl.org/download-2.0.php) has two separate files that must be downloaded
+  1. The `-x86` Runtime Binary (e.g., `SDL2-2.0.12-win32-x86.zip` (as of writing))
+  2. The Visual C++ Development Library (e.g., `SDL2-devel-2.0.12-VC.zip` (as of writing))
+
+For the Runtime Binary, place the extracted `SDL2.dll` into a known folder for later.
+
+- `C:\SDL2\bin\SDL2.dll` will be used as an example
+
+For the Visual C++ Development Library, place the extracted files within a known folder for later.
+
+The following examples will be referenced later:
+
+- `C:\SDL2\lib\x86\*`
+- `C:\SDL2\include\*`
+
+#### Gnuwin
+
+Ensure that this is in %PATH%.
+
+If errors arise (i.e., particularly with the `CREATE_PROCESS('usr/bin/mkdir')` calls, also verify that Git for Windows has not been installed with full Linux support. If it has, remove `C:\Program Files\Git\usr\bin` from the SYSTEM %PATH% until after compilation.
```

Comment:
Can you elaborate about this error? Might be better to actually solve it instead.

Reply:
The Git for Windows error, or the `Error -1073741819`?  

For Git for Windows: if you have Git For Windows installed with "Use Git and optional Unix tools from the Command Prompt" ([see SS here](https://i.imgur.com/TMvAojy.png)) selected, a directory containing additional Linux CLI tools will be added to your `SYSTEM PATH` (default: `C:\Program Files\Git\usr\bin`) which contains executables that I believe either trick the installation process into doing Linux-y things on Windows or causes conflicts with default Windows executables (such as `mkdir`). The following error is produced:  

```
process_begin: CreateProcess(NULL, /usr/bin/mkdir -p build/obj/SDL/audio/, ...) failed.
make (e=2): The system cannot find the file specified.
```

However, if you meant the other error:  

> make: *** [build/bin/BootROMs/dmg_boot.bin] Error -1073741819  

This error sometimes occurs during the linking process (typically just before a lot of `cp`s. It is not always reproducible. I have compiled SameBoy about 60-80 times while making edits for some game hacking last week, and I'd say that I ran into this error roughly 70-90% of the time. Simply running `make` a second time solved the issue all-but-1 times, That one final time, I had to run `make` two additional times (3 total).

Reply:
Aha, it looks like it defaults to the "git" version of `which`, which returns Unix paths instead of Windows paths.